core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -120,6 +100,50 @@ var LearnerDashboardPage = function() {
     expect(await explorationTitle.getText()).toMatch(title);
   };
 
+  this.expectTitleOfLearnerTopicSummaryTileToMatch = async function(title) {
+    var explorationTitle = element(
+      by.cssContainingText(
+        '.protractor-test-learner-topic-summary-tile-title', title));
+    expect(await explorationTitle.getText()).toMatch(title);
+  };
+
+  this.expectNameOfTopicInEditGoalsToMatch = async function(name) {
+    var topicName = element(by.cssContainingText(
+      '.protractor-test-topic-name-in-edit-goals', name));
+    expect(await topicName.getText()).toMatch(name);
+  };
+
+  this.expectNameOfTopicInCurrentGoalsToMatch = async function(name) {
+    await waitFor.visibilityOf(
+      currentGoalsTopic,
+      'Topic in Current Goals takes too long to appear');
+    await waitFor.textToBePresentInElement(
+      currentGoalsTopic, name, 'No Current goals Text');
+    var topicName = element(by.cssContainingText(
+      '.protractor-test-topic-name-in-current-goals', name));
+    expect(await topicName.getText()).toMatch(name);
+  };
+
+  this.expectNameOfTopicInContinueWhereYouLeftOff = async function(name) {
+    await waitFor.visibilityOf(
+      continueWhereYouLeftOffTopicName,
+      'Topic in Continue where you left off takes too long to appear');
+    await waitFor.textToBePresentInElement(
+      continueWhereYouLeftOffTopicName, name,
+      'No topic in Continue where you left off section');
+    var topicName = element(by.cssContainingText(
+      '.protractor-test-topic-name-continue-where-you-left-off'
+    ));
+    expect(await topicName.getText()).toMatch(name);
+  };
+
+  this.addTopicToLearnerGoals = async function() {
+    await waitFor.visibilityOf(
```

Comment:
Please remove this. `action.click` checks for visibility under its hood.

Reply:
Done in #13518.